services/src/handlers/layers.rs

```diff
@@ -455,3 +464,391 @@ async fn layer_handler<C: Context>(
 
     Ok(web::Json(collection))
 }
+
+#[derive(Debug, Deserialize, ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct AddLayerRequest {
+    pub layer: AddLayer,
+    #[schema(value_type = String, example = "05102bb3-a855-4a37-8a8a-30026a91fef1")]
+    pub collection_id: LayerCollectionId,
```

Comment:
I would put the collection in the resource name (url). "POST /layers/collections/{collectionId}/layers" and then put the layerId in the body. Or mayber even put the layer in the url?

Reply:
[ef718cf](https://github.com/geo-engine/geoengine/pull/676/commits/ef718cf9a8feb4115de82d1bf10f2f6992e39854)

Reply:
shouldn't the collection id be assigned by the backend rather than specified? At the least, it should be optional.

Reply:
so what about "POST /layerDb/collections/{collectionId}/layers"?

Reply:
[b520955](https://github.com/geo-engine/geoengine/pull/676/commits/b520955c27aacc7e7167456a52bae5a29661ac25)